tests/examples/test_examples.py

```diff
@@ -198,7 +201,9 @@ def _assert_snapshot(example, url, example_type):
 
     assert success, "%s failed to load" % example.relpath
     assert no_resources, "%s failed with %d missing resources" % (example.relpath, len(resources))
-    assert no_errors, "%s failed with %d errors" % (example.relpath, len(errors))
+
+    if example_type != "server":
```

Comment:
I was able to use this code snippet from the phantomjs docs to wait for slow server responses: https://github.com/ariya/phantomjs/blob/master/examples/waitfor.js

So we won't have to add a blocking sleep in python.